src/Gui/Windows/CombinedCodeViewInteractor.cs

```diff
@@ -299,12 +304,39 @@ public bool ChooseTextEncoding()
             return true;
         }
 
-        private void MixedCodeDataView_MouseDown(object sender, EventArgs e)
+        private void MixedCodeDataView_MouseUp(object sender, MouseEventArgs e)
+        {
```

Comment:
Looks good; once you've addressed my comment below (regarding making the Ctrl+D accelerator specific to the CombinedCodeViewInteractor) I will merge the PR.